hieradata/common.yaml

```diff
@@ -29,6 +29,8 @@ internal_zone_exceptions:
     - dev-death
 
 # IPv4 and IPv6 src ranges for types of servers
+ocf_ipv4_mask: 169.229.226.0/24
+ocf_ipv6_mask: 2607:f140:8801::/48
```

Comment:
nice, we should probably look for other places where we can use this (in another PR)